components/InviteAdminModal/InviteAdminModal.module.css

```diff
@@ -0,0 +1,96 @@
+.button {
+  background-color: #cecccc;
+  color: #646464;
+  border-radius: 7px;
+  height: 33px;
+  font-size: 14px;
+  font-family: Avenir;
+  font-weight: 800;
+  align-self: center;
+  margin-left: auto;
+}
+
+.row {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-around;
+  flex-wrap: wrap;
+  padding-bottom: 15px;
+  /* align-items: baseline; */
+}
+
+.col {
+  width: 40%;
+}
+
+.form {
+  display: flex;
+  /* justify-content: space-evenly; */
+  flex-direction: column;
+}
+
+.modal {
+  /* width: 100%;
+  display: flex; */
+  border-radius: 20px;
+  height: 470px;
+  width: 750px;
+  padding-bottom: 2%;
+  padding-right: 2%;
+  padding-left: 1%;
+}
+
+.cancel {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-width: 70px;
+  height: 30px;
+  border: 1px solid #C8C8C8;
+  border-radius: 16.5px;
+  background-color: var(--off-white);
+  margin-right: 10px;
+  font-size: 14px;
+  font-family: Avenir;
+  /* font-weight: 800; */
+  color: var(--grey-darker);
+}
+
+.actions {
+  margin-top: auto;
+}
+
+.title {
+  display: flex;
+  padding-top: 4%;
+  padding-left: 0%;
+  margin-left: 30px;
+}
+
+.send {
```

Comment:
combine these two in into one CSS rule if you think they're gonna be the same. rename to something like `.modalButtons`